assets/sass/patterns/molecules/investor-logos.scss

```diff
@@ -1,63 +1,62 @@
 @import "../../definitions";
 
+$image-height: 72;
+$image-width: 185;
+
 .investor-logos {
-  width: 100%;
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
 }
 
-.investor-logos__img_container {
+.investor-logos__item {
   border-top: 1px solid $color-text-dividers;
-  @include height(120);
+  @include blg-spacing("top", "small");
+  @include blg-spacing("bottom", "small");
+  @include padding(24, "left");
+  @include padding(24, "right");
```

Comment:
Won't do it here, but it would probably be a good idea to add `vertical` and `horizontal` as options.